packages/xrpl/package.json

```diff
@@ -33,9 +33,6 @@
   "devDependencies": {
     "@geut/browser-node-core": "^2.0.13",
     "@types/node": "^16.18.38",
-    "browserify-fs": "^1.0.0",
```

Comment:
Hopefully soon we can eliminate `node-polyfill-webpack-plugin` which will remove even more dependencies from `package-lock.json`

Reply:
actually we can get rid of `node-polyfill-webpack-plugin` now.  Will eliminate 22 more deps.